src/renderer/atlas/atlas.vcxproj

```diff
@@ -7,10 +7,12 @@
     <ProjectName>RendererAtlas</ProjectName>
     <TargetName>ConRenderAtlas</TargetName>
     <ConfigurationType>StaticLibrary</ConfigurationType>
+    <IncludePath>$(SolutionDir)\dep\DirectXTK\Inc;$(IncludePath)</IncludePath>
   </PropertyGroup>
   <Import Project="$(SolutionDir)src\common.build.pre.props" />
   <Import Project="$(SolutionDir)src\common.nugetversions.props" />
   <ItemGroup>
+    <ClCompile Include=".$(SolutionDir)\dep\DirectXTK\Src\WICTextureLoader.cpp" />

```

Comment:
There's an extra dot here. I think this is breaking the build?

Reply:
Will fix. Been away for a week so slow response from me

Reply:
Fixed. Let's see if it helps